web/app/components/header/account-setting/model-provider-page/model-parameter-modal/index.tsx

```diff
@@ -152,7 +152,7 @@ const ModelParameterModal: FC<ModelParameterModalProps> = ({
     <PortalToFollowElem
       open={open}
       onOpenChange={setOpen}
-      placement='bottom-end'
+      placement='left'
```

Comment:
The model parameter now properly stays on the left.

https://github.com/langgenius/dify/assets/18554747/73fe4d36-cd82-4cb5-bbad-f6c7ea5de828